hlibsass-0.1.8.0 build failure

```
    [1 of 2] Compiling Main             ( /var/stackage/work/unpack-dir/unpacked/hlibsass-0.1.8.0-c91395d719508f34af2d634ba2921de02b67a66044d1fb8edb17cddb37d41482/Setup.hs, /var/stackage/work/unpack-dir/unpacked/hlibsass-0.1.8.0-c91395d719508f34af2d634ba2921de02b67a66044d1fb8edb17cddb37d41482/.stack-work/dist/x86_64-linux/Cabal-3.0.0.0/setup/Main.o )

    /var/stackage/work/unpack-dir/unpacked/hlibsass-0.1.8.0-c91395d719508f34af2d634ba2921de02b67a66044d1fb8edb17cddb37d41482/Setup.hs:41:13: error:
        Not in scope: ‘preSDist’
       |
    41 |           , preSDist = updateLibsassVersion
       |             ^^^^^^^^
```

It is not yet clear to me if this is a stack-related build failure, or if this is a general build failure.

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -2228,8 +2228,8 @@ packages:
         - bytestring-to-vector
 
     "Jakub Fijałkowski <kuba@codinginfinity.me> @jakubfijalkowski":
-        - hlibsass < 0 # https://github.com/commercialhaskell/stackage/issues/5075
-        - hsass < 0 # via hlibsass
+        - hlibsass
+        - hsass
 
     "Robert Massaioli <robertmassaioli@gmail.com> @robertmassaioli":
         - range

